src/components/books/books.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable no-unused-vars */
+function Books({
```

Comment:
- [ ] Please, use ES6 best practices, in this case, this could improve being an arrow function. 🦊